src/lib/config.ts

```diff
@@ -238,7 +239,7 @@ export async function createConfig(options: PreviewOptions = {}): Promise<Previe
     }
   }
 
-  let emote = PreviewEmote.IDLE
+  let emote = null
```

Comment:
If the emote is null as default, no emote will be rendered if there's no emote provided. This might be a kind of "breaking change", we might need to check if we need to change something in the preview in the builder or if we need to communicate to DG that is using the wearable preview to see if they need to set a default emote.

Reply:
Updated so it doesn't include any breaking changes!